rust/xaynet-mobile/src/participant.rs

```diff
@@ -133,6 +142,8 @@ pub struct Participant {
     store: Store,
     /// Async runtime to execute the state machine
     runtime: Runtime,
+    /// Xaynet client
+    client: Client<reqwest::Client>,
```

Comment:
Should that be better an `Arc`?

Reply:
`reqwest::Client` is already an `Arc`, internally, so I think it's fine.

```rust
#[derive(Clone)]
pub struct Client {
    inner: Arc<ClientRef>,
}
```